src/GraphQL/Types/Schema.cs

```diff
@@ -466,6 +475,34 @@ IEnumerable<ISchemaNodeVisitor> GetVisitors()
         {
             // At this point, Initialized will return false, and Initialize will still lock while waiting for initialization to complete.
             // However, AllTypes and similar properties will return a reference to SchemaTypes without waiting for a lock.
+
+            // Wrap resolvers with context accessor if configured
+            if (ResolveFieldContextAccessor != null)
```

Comment:
The code here is what enables the resolve field context accessor functionality.  It is essentially a schema visitor, looking for fields of object graph types, and wrapping their resolvers.  It skips default resolvers, as they do not require the context accessor.  Then the middleware is applied as it always has been, which gets applied over (before) the resolver runs.  We can't just add schema-level middleware to do this, as schema-level middleware is always called before field-level middleware (with this PR).  I could rewrite this code in a separate method for organization, or within a schema visitor for clarity.

Reply:
I think putting it in a schema visitor is cleaner